### PR TITLE
Remove superfluous option, and move a default to the environment.

### PR DESCRIPTION
**Pull request by andreas-kupries**

### CODE REVIEW

File: make/kube [hunk 1]
```diff
@@ -6,24 +6,24 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
...
 
 if [ "${1:-}" = "helm" ]; then
     CREATE_HELM_CHART=true
     BUILD_TARGET=helm
-    ENV_FILES="$(echo bin/settings/certs.env bin/settings/kube/ca.env)"
 fi
 
-rm -rf "${BUILD_TARGET}"
+if [ "${BUILD_TARGET}" = "kube" ]; then
+    # Ovverides when generating kube config files instead of helm charts.
```
Comment (alexjh) [inline make/kube]: s/Ovverides/Overrides

Comment (andreas-kupries) [inline make/kube]: Fixed in commit bd5a1a19004879c4fc541c0bbfba350a6125abca

File: make/kube [hunk 2]
```diff
@@ -6,24 +6,24 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
...
     BUILD_TARGET=helm
-    ENV_FILES="$(echo bin/settings/certs.env bin/settings/kube/ca.env)"
 fi
 
-rm -rf "${BUILD_TARGET}"
+if [ "${BUILD_TARGET}" = "kube" ]; then
+    # Overrides when generating kube config files instead of helm charts.
+    FISSILE_OUTPUT_DIR="${PWD}/kube"
+    ENV_FILES="$(echo bin/settings/*.env bin/settings/kube/*.env)"
+    FISSILE_DEFAULTS_FILE="$(echo "${ENV_FILES}" | tr ' ' ,)"
```
Comment (alexjh) [inline make/kube]: I don't think this is having any effect, shouldn't it be exported?

Comment (andreas-kupries) [inline make/kube]: Ah. Likely yes. Will fix.

Comment (andreas-kupries) [inline make/kube]: Done with commit 6191fcb39b295af47166c4b353544c0e5f3da746

**Pull request merged**